edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.Test/DockerConfigTest.cs

```diff
@@ -187,20 +202,49 @@ public void TestSerializationDigest()
         }
 
         [Theory]
-        [InlineData(null, null, typeof(ArgumentException))]
-        [InlineData("", null, typeof(ArgumentException))]
-        [InlineData("  ", null, typeof(ArgumentException))]
-        [InlineData("mcr.ms.com/ea:t1:t2", null, typeof(ArgumentException))]
```

Comment:
why remove these tests?

Reply:
Oh I see, you just moved them..

Reply:
I just moved them under the headings. I can remove the headings if that is preferred.